fluid/mnist/_ce.py

```diff
@@ -0,0 +1,61 @@
+# this file is only used for continuous evaluation test!
+
+import os
+import sys
+sys.path.append(os.environ['ceroot'])
+from kpi import CostKpi, DurationKpi, AccKpi
```

Comment:
when will the kpi.py available? @Superjomn

Reply:
the kpi.py file will be copyed from ce framework， when begin to run the model.